runtime/src/accounts.rs

```diff
@@ -1793,4 +1812,46 @@ mod tests {
         info!("done..cleaning..");
         accounts.accounts_db.clean_accounts();
     }
+
+    fn load_accounts_no_store(
```

Comment:
Must be missing something, where is this used?

Reply:
It's a helper for the `test_instructions` test case which loads the special instructions account.

https://github.com/solana-labs/solana/pull/11943/files#diff-0104dfa96426a3a49428815872a0356eR1839